### PR TITLE
Remove OutputDebugStringA wrapper on Linux, simplify PRINT_VAR and PRINT macros.

### PR DESCRIPTION
**Pull request by pierricgimmig**
- move PRINT from PrintVar to OrbitBase/Logging.h
    - add PRINTF to simplify printing raw formatted strings (not using LOG)
    - Add Windows.h include in OrbitBase/Logging.h for OutputDebugStringA
    - Fix compilation error due to not including Windows.h in OrbitBase/Loggins.h

### CODE REVIEW

File: OrbitBase/include/OrbitBase/Logging.h [hunk 1]
```diff
@@ -15,6 +15,13 @@
 #pragma clang diagnostic ignored "-Wgnu-zero-variadic-macro-arguments"
 #endif
 
+#define PRINT(msg) PLATFORM_PRINT(msg)
+
+#define PRINTF(format, ...)                                \
```
Comment (dpallotti) [inline OrbitBase/include/OrbitBase/Logging.h:20]: I fear that this is going to be abused. The way I see it every print should be associated with file and line (and possibly timestamp in the future) and hence every print should use LOG.

File: OrbitBase/include/OrbitBase/Logging.h [hunk 2]
```diff
@@ -73,13 +80,13 @@
...
-#define PLATFORM_LOG(message)       \
+#define PLATFORM_PRINT(message)     \
   do {                              \
     fprintf(stderr, "%s", message); \
     OutputDebugStringA(message);    \
   } while (0)
 #else
-#define PLATFORM_LOG(message)       \
+#define PLATFORM_PRINT(message)     \
   do {                              \
```
Comment (dpallotti) [inline OrbitBase/include/OrbitBase/Logging.h:90]: For just one line that already has to be terminated with `;` the do-while is not necessary.

File: OrbitCore/OrbitFunction.cpp [hunk 3]
```diff
@@ -83,10 +83,10 @@ bool Function::Hookable() {
 void Function::Select() {
   if (Hookable()) {
     selected_ = true;
-    PRINT("Selected %s at 0x%" PRIx64 " (address_=0x%" PRIx64
-          ", load_bias_= 0x%" PRIx64 ", base_address=0x%" PRIx64 ")\n",
-          pretty_name_.c_str(), GetVirtualAddress(), address_, load_bias_,
-          module_base_address_);
+    PRINTF("Selected %s at 0x%" PRIx64 " (address_=0x%" PRIx64
```
Comment (dpallotti) [inline OrbitCore/OrbitFunction.cpp:86]: For example, here, why PRINTF and not LOG?

File: OrbitBase/include/OrbitBase/Logging.h (hunk 1 above)
Comment (pierricgimmig) [reply OrbitBase/include/OrbitBase/Logging.h:20]: Yeah that makes sense.  I'll convert to LOG.

**Comment by pierricgimmig** [comment]
@dpallotti , @antonrohr , thanks for your comments.  I completely removed PRINT and replaced with LOG, also greatly simplified PrintVar.h.  Will close this PR and reopen because I made a mess rebasing...

File: OrbitCore/OrbitFunction.cpp (hunk 3 above)
Comment (pierricgimmig) [reply OrbitCore/OrbitFunction.cpp]: Done.

File: OrbitBase/include/OrbitBase/Logging.h (hunk 2 above)
Comment (pierricgimmig) [reply OrbitBase/include/OrbitBase/Logging.h]: Done.

**Comment by pierricgimmig** [comment]
Cleaned up PR in https://github.com/google/orbit/pull/393

**Pull request closed without merging**